### DISCUSSION TITLE
Miniaturas Vista Previa en klipper  - Solución de Novato - flsun v400

### DISCUSSION THREAD

**Discussion by Fenacor** [OrcaSlicer/OrcaSlicer#4361] [General]
Desde que opte con el Orca para la Flsun V400 solo me ha molestado algo, el no poder ver las miniaturas de los gcode

![Captura de pantalla 2024-03-06 214621](https://github.com/SoftFever/OrcaSlicer/assets/152562828/5a005927-14b7-4118-b24b-9b33914237bb)

básicamente si tiene la opción pero no se veía a pesar de haber configurado varias veces
![Captura de pantalla 2024-03-06 214759](https://github.com/SoftFever/OrcaSlicer/assets/152562828/a35b0458-cc2a-4c97-9fb8-ae19594c7585)

>  (800x800) es perfectamente soportado por el speederpad de flsun.

Luego de básicamente haber aceptado que no se podía y haber visto varios tutoriales y no logrado nada.

Debo mencionar que el fallo no es del orca ya que al hacer el corte si incrusta la miniatura, es el klipper el que no lo entiende.

Me decidí a comparar un gcode al cual si se le veía la miniatura con un gcode cortado con el orca y luego de un poco de ensayo y error 
### **¡finalmente puede ver la miniatura de un nuevo!.**

![Captura de pantalla 2024-03-06 214742](https://github.com/SoftFever/OrcaSlicer/assets/152562828/1aa629ec-f995-4e9c-869f-83772b791136)

Después de eso fui quitando líneas de código hasta ver que lo único que requiera era esta.
![Captura de pantalla 2sssss024-03-06 214736](https://github.com/SoftFever/OrcaSlicer/assets/152562828/04ab898c-8033-463e-8146-06796847ff51)

`;Generated with Cura_SteamEngine 5.6.0 `  <---agrega al gcode

> Supongo es una configuración en el speeder pad 

Y para que ya funcione de ahora en adelante solo faltaría agregarlo al gcódigo inicial en la impresora del orca así al hacer un corte automáticamente lo agrega al modelo (al parecer no afecta donde lo pongas)

quedando de esta manera
`;Generated with Cura_SteamEngine 5.6.0   
G21
G90
M82
M107 T0
M140 S[bed_temperature_initial_layer_single]
M104 S[nozzle_temperature_initial_layer] T0
M190 S[bed_temperature_initial_layer_single]
M109 S[nozzle_temperature_initial_layer] T0
G28
G1 F3000 Z1
G1 X-150 Y0 Z0.4
G92 E0
G3 X0 Y-130 I150 Z0.3 E30 F2000
G92 E0` 

> este es el código inicial de fabrica del flsun v400 con el agregado anterior

![Captura de pantalla 2024-03-06 221822](https://github.com/SoftFever/OrcaSlicer/assets/152562828/8d6428c0-b383-4488-9e8b-156c1110b2de)
![Captura de pantalla 2024-03-06 221843](https://github.com/SoftFever/OrcaSlicer/assets/152562828/95589fd9-16c3-4552-b53e-ddd2ec15d8d6)

![Captura de pantalla 2024-03-06 224141](https://github.com/SoftFever/OrcaSlicer/assets/152562828/9aa60bec-0645-4579-813f-bc2fa34a2882)

**Comment by avanegas2021** [comment]
Gracias, estuve buscando esta solución por un tiempo, luego me habia resignado a que no tenia las miniaturas en ORCA. Ahora buscando la manera en que recupere la impresión luego de un corte de luz, encontré tu post. muy util

**Reply by Fenacor** [reply]
Que bueno que sea útil el tema, si llegas a encontrar lo de la luz seria muy util, en mi primer corte de luz compre nobreak de 600va y bueno eso fue todo

**Comment by logandk22** [comment]
un agregado al tema, ademas de agregar el  codigo  ";Generated with Cura_SteamEngine 5.6.0" tambien es necesacio modificar en la opcion "Minuaturas de G-Code 160x160/PNG;320x320/PNG " borrar la opcion de 320X320 ya que en mi caso no mejaba ver la imagen si estan las 2 opciones
<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/913771a1-b69f-479e-b4f9-8d1ad5b85be1" />
<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/648a5a86-1ddf-4709-9795-015ad9c0953a" />

**Comment by GustavoMM11** [comment]
Esta opción si me funcionó!!! gracias
